Weather Today javascript app

ajkacca457 opened the pull request:
In this project  :
- I have used webpack to bundle different javascript files.
- all the dom elements are created dynamically using javascript.
- Worked with the API of OpenWeather.
- Used async and await with fetch promise.

trekab commented on package.json:
```diff
@@ -0,0 +1,38 @@
+    "webpack": "^5.10.3",
+    "webpack-cli": "^4.2.0"
+  },
+  "scripts": {
+    "build":"webpack",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Add the necessary commands like `build`, `development` in the `package.json` inside __this__ `scripts` section. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/scripts_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/))

trekab reviewed (changes requested):
### Changes Required! :recycle: 
Hi @ajkacca457! Great job with the UI! :+1: I like the extra country flag image displayed about the location. However, there are some details for you to work on to finish up this project. These include:
- [x] Add a toggle control to determine if the result will be shown in Celsius or Fahrenheit. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [x] Try to make your commit messages more descriptive eg. `initial webpack setup` could be better expressed as `Add initial webpack setup`. [Here](https://hackwild.com/article/semantic-git-commits/) is a good guide for writing better commit messages. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_commits.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)) __[Note]__ _You don't need to edit your commit messages. Just write better ones from now on._
- [ ] __[Optional]__ Add a ‘loading’ component that displays from the time the form is submitted until the information comes back from the API ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/loading_component_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app))
- [ ] __[Optional]__ Make your app fetch the current geolocation of the user when the app is first accessed and let it fetch the weather from that location.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink: 
__Treasure Kabareebe__
[Portfolio](https://trekab.github.io/portfolio/) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

ajkacca457 commented:
@trekab thank you for your constructive feedback. Based on your feedback my project looks and works way better. I have added the Celsius to Fahrenheit and vice versa functionally. I have added loader functionality when the user will submit the form. I will be careful about my commit messages from now. Hopefully, now my project is acceptable. Thanks

rOluochKe reviewed (approved):
Hello @ajkacca457 
### Well done, you got it right :clap: The project has been approved. Congratulations :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!
